pkg/ecdsa/dkg/member.go

```diff
@@ -126,3 +170,27 @@ func (fm *finalizingMember) Result() *Result {
 		SymmetricKeys: fm.symmetricKeys,
 	}
 }
+
+func generateTssPartiesIDs(
+	memberID group.MemberIndex,
+	groupMembersIDs []group.MemberIndex,
+) (*tss.PartyID, []*tss.PartyID) {
+	var partyID *tss.PartyID
+	var groupPartiesIDs []*tss.PartyID
+
+	for _, groupMemberID := range groupMembersIDs {
+		newPartyID := tss.NewPartyID(
+			strconv.Itoa(int(groupMemberID)),
+			"",
+			big.NewInt(int64(groupMemberID)),
+		)
+
+		if memberID == groupMemberID {
+			partyID = newPartyID
+		}
+
+		groupPartiesIDs = append(groupPartiesIDs, newPartyID)
+	}
```

Comment:
since we already know the size of size of `groupPartiesIDs` ahead of time; it will be equal to `len(groupMembersIDs)`, we can save ourselves some allocations:

option 1:
```go
groupPartiesIDs := make([]*tss.PartyID, 0, len(groupMembersIDs))
```

or more efficiently but uglier IMO
```go
	groupPartiesIDs := make([]*tss.PartyID, len(groupMembersIDs))

	for i, groupMemberID := range groupMembersIDs {
		newPartyID := tss.NewPartyID(
			strconv.Itoa(int(groupMemberID)),
			"",
			big.NewInt(int64(groupMemberID)),
		)

		if memberID == groupMemberID {
			partyID = newPartyID
		}

		groupPartiesIDs[i] = newPartyID
	}
```

Reply:
That's right! See e0d8501e